service/src/main/java/com/codeforcommunity/processor/MapProcessorImpl.java

```diff
@@ -1,15 +1,17 @@
 package com.codeforcommunity.processor;
 
-import static org.jooq.generated.Tables.BLOCKS;
-import static org.jooq.generated.Tables.NEIGHBORHOODS;
-import static org.jooq.generated.Tables.RESERVATIONS;
+import static org.jooq.generated.Tables.*;
```

Comment:
Can you change this back to singular imports?

Reply:
Are singular imports always preferred? It seems IntelliJ combines them after it gets to five or so